spyderlib/plugins/shortcuts.py

```diff
@@ -159,17 +63,12 @@ def data(self, index, role=Qt.DisplayRole):
         column = index.column()
         if role == Qt.DisplayRole:
             if column == CONTEXT:
-                return to_qvariant(shortcut.context)
+                return to_qvariant(_(shortcut.context.capitalize()))

```

Comment:
Yep, I think is better for another PR, plus not sure what would be the best way @ccordoba12 to implement it...